requires-dev.txt

```diff
@@ -1,14 +1,15 @@
 dash_flow_example==0.0.5
 dash-dangerously-set-inner-html
-isort==4.3.21
-mock==4.0.1;python_version>="3.0"
+isort==4.3.21;python_version<"3.7"
+mock==4.0.3;python_version>="3.0"
 mock==3.0.5;python_version=="2.7"
-flake8==3.7.9
-PyYAML==5.3
-pylint==1.9.4;python_version<"3.7"
-pylint==2.3.1;python_version=="3.7"
-astroid==2.2.5;python_version=="3.7"
+flake8==3.8.4
+PyYAML==5.3.1
+pylint==1.9.5;python_version<"3.7"
+pylint==2.6.0;python_version>="3.7"
```

Comment:
Could we sanity test these flake8/pylint changes against the other core repos?

Reply:
They have an annoying habit of changing their rules..

Reply:
Good call - they may well need their own updates to `.pylintrc` files, or syntax updates. PRs coming.

Reply:
https://github.com/plotly/dash-core-components/pull/907, https://github.com/plotly/dash-html-components/pull/171, https://github.com/plotly/dash-table/pull/857 - All three, I started with this branch, got tests passing, then reverted to the dev branch of dash and tests still pass - All that was required (other than prohibiting the latest `xlrd`) was a couple of linting exclusions.